components/authz-service/server/v2/policy_test.go

```diff
@@ -1174,7 +1174,7 @@ func TestReplacePolicyMembers(t *testing.T) {
 
 			require.Nil(t, pol)
 			grpctest.AssertCode(t, codes.InvalidArgument, err)
-			assert.Regexp(t, "invalid ReplacePolicyMembersReq.Id: value does not match regex pattern", err.Error())
+			assert.Regexp(t, "rpc error: code = InvalidArgument desc = a policy id is required and must contain at least one non-whitespace character", err.Error())
```

Comment:
No need for `assert.Regexp` if you are matching exactly... however, that is rather fragile. Would we, for example, want to fail, if the generator produced "code=InvalidArgument" instead of "code = InvalidArgument" ? Consider matching just the key content:
```suggestion
			assert.Regexp(t, "InvalidArgument.*policy id is required.*must contain at least one non-whitespace character", err.Error())
```